operator/pkg/release/lookup.go

```diff
@@ -99,9 +99,9 @@ func (c *channelConfigMapLookup) Lookup(ctx context.Context) (*corev1.ConfigMap,
 	}
 
 	if actualChannel != c.channel {
-		log.FromContext(ctx).Info(fmt.Sprintf("using %s (instead of %s) for component %s", actualChannel, c.channel, c.component.Name))
+		log.FromContext(ctx).V(0).Info(fmt.Sprintf("using %s (instead of %s) for component %s", actualChannel, c.channel, c.component.Name))
 	} else {
-		log.FromContext(ctx).Info(fmt.Sprintf("using %s for component %s", actualChannel, c.component.Name))
+		log.FromContext(ctx).V(0).Info(fmt.Sprintf("using %s for component %s", actualChannel, c.component.Name))
```

Comment:
isnt't verbosity level by default 0?

Reply:
True, left it by accident from a bit of debugging, Ill remove it